lib/task-data/schemas/dell-racadm-catalog.json

```diff
@@ -1,16 +1,16 @@
 {
-    "copyright": "Copyright 2016, EMC, Inc.",
-    "definitions": {
-       "action": {
-            "description": "Action for catalog related tasks, currently only getConfigCatalog is supported",
-            "type": "string",
-            "enum": ["getConfigCatalog"]
-        }
-    },
-    "properties": {
-        "action": {
-            "$ref": "#/definitions/action"
-        }
-    },
-    "required": ["action"]
+  "copyright": "Copyright 2016, EMC, Inc.",
+  "definitions": {
+    "action": {
+      "description": "Action for catalog related tasks, currently only getConfigCatalog is supported",
```

Comment:
The description need be updated, as `getConfigCatalog` is not the only one to be supported now.